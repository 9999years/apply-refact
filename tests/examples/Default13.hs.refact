[("tests/examples/Default13.hs:1:7-24: Suggestion: Use <$>\nFound:\n  a >>= return . bob\nPerhaps:\n  bob <$> a\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 7, endLine = 1, endCol = 25}, subts = [("f",SrcSpan {startLine = 1, startCol = 22, endLine = 1, endCol = 25}),("m",SrcSpan {startLine = 1, startCol = 7, endLine = 1, endCol = 8})], orig = "f <$> m"}])]
