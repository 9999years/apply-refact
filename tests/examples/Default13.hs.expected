yes = Control.Monad.liftM bob a
